src/utilities/LRUCache.js

```diff
@@ -135,6 +145,28 @@ class LRUCache {
 
 	}
 
+	// Marks whether tiles in the cache have been completely loaded or not. Tiles that have not been completely
+	// loaded are subject to being disposed early even if the are marked as used if the cache is full above its
+	// max size limits.
```

Comment:
nit There is a word missing here

Reply:
the test check is a good addition :+1:

Reply:
Thanks - just reworded it. Let me know once you think everything looks good and we can get this merged and a new version published.